<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ESPACE CLIENT</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>601</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="widget">
    <property name="font">
     <font>
      <family>Sitka Text</family>
     </font>
    </property>
    <attribute name="title">
     <string>Les légumes en stock</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Banner</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Traditionnel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bio</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sitka Text</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hiver</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Printemps</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Eté</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Automne</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>19</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Choisir la saison:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Print</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Choisir la catégorie: </string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>111</y>
       <width>1011</width>
       <height>441</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>26</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="afficher">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe Script</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Afficher</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Passer une commande</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>381</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Entrez le nom de légume:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Quantité désiré:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>129</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>28</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>391</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Numéro de carte bancaire:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>300</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Centaur</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>410</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Passer la commande</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
